.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### Changelist
+[Describe or list the changes made in this PR]
+
+### Test Plan
+[Describe how this PR was tested (if applicable)]
+
+### Author/Reviewer Checklist
+- Add an entry to `protocol/CHANGELOG.md` and/or `indexer/CHANGELOG.md` (if applicable).
+- If this PR has changes that modify the way state is read or written, require state migration, or change app hash, manually add the `state-breaking` label.
```

Comment:
As a follow up item, I think it would be good to explain what qualifies as state-breaking with examples

Reply:
I put a better description. I'm not quite sure what to do for examples for now because there are many ways things can change the app state, and an example might not help people know what to look for. Let's think about this later.

Reply:
I think picking recent examples are fine for now. Again, this won't be comprehensive but gives ppl ideas why certain changes are state-breaking.